### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by KaoImin**
**What this PR does / why we need it**:

This PR

**Which issue(s) this PR fixes**:

Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

NIL

**CI Description**

| CI Name                                   | Description                                                     |
| ----------------------------------------- | --------------------------------------------------------------- |
| *Chaos CI*                                | Test the liveness and robustness of Axon under terrible network condition    |
| *Cargo Clippy*                            | Run `cargo clippy --all --all-targets --all-features`      |
| *Coverage Test*                           | Get the unit test coverage report                             |
| *E2E Test*                                | Run end-to-end test to check interfaces                         |
| *Code Format*                             | Run `cargo +nightly fmt --all -- --check` and `cargo sort -gwc`     |
| *Web3 Compatible Test*                    | Test the Web3 compatibility of Axon                               |
| *v3 Core Test*                            | Run the compatibility tests provided by Uniswap V3             |
| *OCT 1-5 \| 6-10 \| 11 \| 12-15 \| 16-19* | Run the compatibility tests provided by OpenZeppelin           |

**CI Usage**

> Check the CI you want to run below, and then comment `\run-ci`.

**CI Switch**

- [ ] Chaos CI
- [ ] Cargo Clippy
- [ ] Coverage Test
- [ ] E2E Tests
- [ ] Code Format
- [ ] Unit Tests
- [ ] Web3 Compatible Tests
- [ ] OCT 1-5 And 12-15
- [ ] OCT 6-10
- [ ] OCT 11
- [ ] OCT 16-19
- [ ] v3 Core Tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +36,8 @@ Most of the infrastructure has been done and some substantial features to be dev
 
 1. Be compatible with [IBC](https://github.com/cosmos/ibc) protocol.
 2. Implement the original homologous cross-chain protocol.
-3. Implement more cross-chain protocol.
+3. Implement a cross-chain protocol with EVM-based chain.
```
Comment (linnnsss) [inline README.md]: 3. Implement a cross-chain protocol with EVM-based chains.

File: README.md [hunk 2]
```diff
@@ -37,7 +36,8 @@ Most of the infrastructure has been done and some substantial features to be dev
...
+4. Implement more cross-chain protocol.
```
Comment (linnnsss) [inline README.md]: 4. Implement more cross-chain protocols.

**Review by kelisi129 — APPROVED** [review]

**Review by linnnsss — APPROVED** [review]

**Review by linnnsss — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -37,7 +36,8 @@ Most of the infrastructure has been done and some substantial features to be dev
 
 1. Be compatible with [IBC](https://github.com/cosmos/ibc) protocol.
 2. Implement the original homologous cross-chain protocol.
-3. Implement more cross-chain protocol.
+3. Implement a cross-chain protocol with EVM-based chains.
+4. Implement more cross-chain protocols.
```
Comment (kelisi129) [inline README.md]: Compatible with more cross chain protocols in the future.

File: README.md [hunk 4]
```diff
@@ -37,7 +36,8 @@ Most of the infrastructure has been done and some substantial features to be dev
 
 1. Be compatible with [IBC](https://github.com/cosmos/ibc) protocol.
 2. Implement the original homologous cross-chain protocol.
-3. Implement more cross-chain protocol.
+3. Implement a cross-chain protocol with EVM-based chains.
```
Comment (kelisi129) [inline README.md]: Design a cross-chain protocol for EVM-based chains.

**Review by kelisi129 — CHANGES_REQUESTED** [review]

**Review by kelisi129 — APPROVED** [review]

**Pull request merged**